ethcore/native_contracts/build.rs

```diff
@@ -43,10 +45,16 @@ fn build_file(name: &str, abi: &str, filename: &str) {
 	f.write_all(code.as_bytes()).unwrap();
 }
 
+fn build_test_contracts() {
```

Comment:
(IMO I don't think it's worth doing since this crate barely ever needs to be recompiled)